automation/cypress/support/step_definitions/home.step.js

```diff
@@ -41,10 +41,10 @@ Then("the vesting address is shown", () => {
 });
 
 Then("the total supply is shown correctly", () => {
-  cy.get('[data-testid="total-supply"]').should(
-    "have.text",
-    "1,000,000,000.00"
-  );
+  cy.get('[data-testid="total-supply"]').then(($supply) => {
+    const supplyNum = Number($supply.text().replace(/,/g, ""));
+    expect(supplyNum).to.be.above(65000000);
```

Comment:
noice !!!

Reply:
This is wrong, there can be fewer tha 65M tokens. As you can see on the live mainnet token [here](https://etherscan.io/token/0xcb84d72e61e383767c4dfeb2d8ff7f4fb89abc6e)

```suggestion
    expect(supplyNum).to.be.above(64999723);
```